### PR TITLE
Add access logging tests

### PR DESCRIPTION
**Pull request by mocenas**
### Summary

Add tests to cover https://issues.redhat.com/browse/QUARKUS-4663

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [X] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [X] Methods and classes used in PR scenarios are meaningful
- [X] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        // check that one request actually match one line in a log file
+        assertEquals(3, Files.lines(accessLogPath()).count(), "There should be 3 lines in access log.");
+
+        assertEquals(1, getLogFileNames().size(), "There should be just one log file");
+    }
+
+    @Test
+    public void logRotationTest() throws IOException {
+        // use zip compression for archived logs
+        setPropertyTo("quarkus.log.handler.file.access-log.rotation.file-suffix", "yyyy-MM-dd.zip");
```
Comment (fedinskiy) [inline logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java:67]: We already have this property in `accessLogging.properties`

Given `gzipCompressionTest`, I suggest we drop this option from the properties file

File: logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        app.stop();
+        app.withProperty(property, value);
+        app.start();
+    }
+
+    private boolean isValidGzipFile(Path file) throws IOException {
+        try (InputStream is = new FileInputStream(file.toFile())) {
+            byte[] signature = new byte[2];
+            int nread = is.read(signature);
+            return nread == 2 && signature[0] == (byte) 0x1f && signature[1] == (byte) 0x8b;
```
Comment (fedinskiy) [inline logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java:127]: A link to specification would be appreciated

**Review by fedinskiy — CHANGES_REQUESTED** [review]
Thank you! Just two minor things

File: logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java (hunk 2 above)
Comment (mocenas) [reply logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java:128]: added

File: logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java (hunk 1 above)
Comment (mocenas) [reply logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/AccessLoggingIT.java:67]: Hmm, OK. I removed it from `accessLogging.properties`. It does not break anything and you're right, this way it is cleaner.

**Review by fedinskiy — APPROVED** [review]

**Pull request merged**